README.md

```diff
@@ -8,25 +8,54 @@ The website for the PySlackers slack community - a place for python learners, te
 
 Prereqs:
 
-* `docker`
-* `python3.6+`
+* docker
+* python3.6+
+
+The following external services are required
+
+* PostgreSQL 9.6+
+* Redis
+
+How you install these services is up to you, but the easiest way is by using
+Docker and Docker compose.
+
+1. Install Docker and Docker Compose by following the instructions on the [Docker website](https://docs.docker.com/compose/install/)
+
+2. Run Docker Compose in daemon mode `-d`:
```

Comment:
Do you think that people will be confused if they research compose and they use the term "detached"? It's the same thing, afaik, but not sure if it could leave confusion.